Script/char.dat

```diff
@@ -3031,7 +3532,68 @@ femaleslave
     {
       2,
       "\"Never say no to Petrus. Ever!\"",
-      "\"And I thought Richel Decos was bad with his whips...\"";
+      "\"And I thought Richel Decos was bad with his whips...\"";
```

Comment:
Her story was changed. I still disagree with your reasoning above, but I like this new story more than the one before, so hopefully you will like it more, too. :P